src/front/app/components/Challenges/Challenge.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+const Challenge = ({ description, user, location }) => (
+    <li className="challenge">
+        <div className="challenge__image">Image here</div>
+        <span className="challenge__description">
+            {description && description.toUpperCase()}
```

Comment:
You mount this component in the map function in its parent synchronously, so this component will not be rendered unless the challenges array in the parent component is not empty. Another point: should such component be rendered if it doesn't have the description and other props?